analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/References.cs

```diff
@@ -24,6 +24,9 @@ namespace SonarAnalyzer.SymbolicExecution.Roslyn.OperationProcessors
 {
     internal static class References
     {
+        public static ProgramState ProcessThis(SymbolicContext context) =>
+            context.State.SetOperationValue(context.Operation, SymbolicValue.This);     // Implicit and Explicit
```

Comment:
Can we have a test for "implicit"?

Reply:
The same concerns about `SetOperationValue` dropping other constraints apply here too. See the comments in https://github.com/SonarSource/sonar-dotnet/pull/5884/files#r925797569

Reply:
Implicit is more difficult to assert, but doable